roles/satellite-clone/tasks/main.yml

```diff
@@ -184,6 +184,15 @@
     state: latest
   when: puppet_version|int == 4
 
+  # Because of https://projects.theforeman.org/issues/30506
+- name: Install Satellite SELinux packages for 6.8 and above
```

Comment:
Is this actually meant to be for 6.8 and above? This would run on all versions. 

If the packages are expected to be present for all versions that is ok, just want to double check that's the case :monocle_face:

Reply:
Technically speaking this is for 6.8 and above but the idea is that there shouldn't be harm in ensuring the packages are present when cloning 6.7 and below (they should already be there due to RPM deps for older versions).

Reply:
sounds good :+1: